### ISSUE TITLE
Mismatches in GradientBoostingRegressor model with lad loss between sklearn and onnx model predictions

### ISSUE THREAD

**Issue by prabhat00155** [onnx/sklearn-onnx#156]

**Comment by prabhat00155** [comment]
X, y = make_regression(n_features=40, n_samples=10000, random_state=42)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)

model = GradientBoostingRegressor(n_estimators=10, random_state=42, loss='lad')
model.fit(X_train, y_train)

onnx_fs = convert_sklearn(model, 'gbr', [('float_input', FloatTensorType(X_test.shape))])

onnxmltools.utils.save_model(onnx_fs, 'gbr.onnx')
sess = InferenceSession('gbr.onnx')
res = sess.run('', input_feed={'float_input': X_test.astype('float32')})

np.mean(np.isclose(model.predict(X_test), list(map(lambda x: x[0], res[0]))))
